### PR TITLE
Fix the prototype nightly deployment pipeline

### PR DESCRIPTION
**Pull request by martyngigg**
**Description of work.**

The new [nightly pipeline](https://builds.mantidproject.org/job/main_nightly_deployment_prototype/) had been hitting problems in passing. This pull request contains a set of fixes that have been tested with that job to get it to a passing state.

**To test:**

* Code review
* See the job is passing the expected pipeline steps

*There is no associated issue.*

*This does not require release notes* because **it is a tool for development/deployment and not visible to users.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by thomashampson — APPROVED** [review]
Pipeline is creating the standalone packages and updating the conda nightly packages.

File: installers/conda/linux/qt.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[Paths]
+Prefix = ./../
```
Comment (sf1919) [inline installers/conda/linux/qt.conf:2]: Should there be a newline at the end of the file?

File: installers/conda/win/create_package.sh [hunk 2]
```diff
@@ -215,11 +222,4 @@ echo makensis /V4 /O\"$NSIS_OUTPUT_LOG\" /DVERSION=$VERSION /DPACKAGE_DIR=\"$COP
...
 echo "Package packaged, find it here: $OUTFILE_NAME"
 
-echo "Cleaning up left over files"
-rm -rf $CONDA_ENV_PATH
-rm -rf $COPY_DIR
-rm -rf $NSIS_CONDA_ENV_PATH
-rm $THIS_SCRIPT_DIR/uninstall_files.nsh
-rm $THIS_SCRIPT_DIR/uninstall_dirs.nsh
-
 echo "Done"
```
Comment (sf1919) [inline installers/conda/win/create_package.sh:225]: Same as above - should there be a newline at the end of the file?

**Comment by martyngigg** [comment]
@mantidproject/gatekeepers Could we get this merged for tonight?

**Pull request merged**